### ISSUE TITLE
Masked language modeling?

### ISSUE THREAD

**Issue by serendipity800** [THUDM/ChatGLM-6B#1386]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I can't find anything about this.

### Expected Behavior

Can we use the ChatGLM to train on the masked language modeling task, just like BERT?
Like "[MASK] is the Capital of U.S.", given the correct answer Washington, can we use this to train ChatGLM?

### Steps To Reproduce

1. I don't know

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```
